Cargo.toml

```diff
@@ -35,9 +35,9 @@ digest = { version = "0.8" }
 derivative = { version = "2", features = ["use_core"] }
 
 [dev-dependencies]
-algebra = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl", features = ["bls12_381", "jubjub", "bn_382", "tweedle"]}
-algebra-utils = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl", features = [ "bn_382", "tweedle" ] }
-r1cs-std = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl", features = ["bn_382", "tweedle"]}
+algebra = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl", features = ["tweedle"]}
+algebra-utils = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl", features = [ "tweedle", "bn_382" ] }
```

Comment:
Not sure what you mean. In ginger-lib none of curves enabled by default, so we always need to specify in features needed curve

Reply:
In algebra-utils we used features "tweedle" and "bn_382" because there was Marcelo's code selecting the window size in MSM affine based on the type of the curve, but we removed that piece of code so those features shouldn't be needed anymore

Reply:
Yes, in dependencies these features are not needed anymore. In dev-dependencies "bn_382" feature is not needed, but tweedle is used in all tests. I've removed bn_382 from dev features